### PR TITLE
Update sets.json

### PR DESCRIPTION
**Pull request by DrMabulle**
Insist on the fact that the original Set is not modified

### CODE REVIEW

File: app/json/sets.json [hunk 1]
```diff
@@ -19,9 +19,10 @@
     },
     {
       "preparagraph": "Sets can be added to easily:",
-      "code": "val mySet = Set(\"Michigan\", \"Ohio\", \"Wisconsin\", \"Iowa\")\nval aNewSet = mySet + \"Illinois\"\n\naNewSet.contains(\"Illinois\") should be(__)",
+      "code": "val mySet = Set(\"Michigan\", \"Ohio\", \"Wisconsin\", \"Iowa\")\nval aNewSet = mySet + \"Illinois\"\n\naNewSet.contains(\"Illinois\") should be(__)\n\mySet.contains(\"Illinois\") should be(__)",
```
Comment (rafaparadela) [inline app/json/sets.json]: Format error: Remove the `\` symbol at `\n\mySet...`. Shuld be `\nmySet...`.

File: app/json/sets.json [hunk 2]
```diff
@@ -45,9 +46,10 @@
     },
     {
       "preparagraph": "Set elements can be removed easily:",
-      "code": "val mySet = Set(\"Michigan\", \"Ohio\", \"Wisconsin\", \"Iowa\")\nval aNewSet = mySet - \"Michigan\"\n\naNewSet.contains(\"Michigan\") should be(__)",
+      "code": "val mySet = Set(\"Michigan\", \"Ohio\", \"Wisconsin\", \"Iowa\")\nval aNewSet = mySet - \"Michigan\"\n\naNewSet.contains(\"Michigan\") should be(__)\n\mySet.contains(\"Michigan\") should be(__)",
```
Comment (rafaparadela) [inline app/json/sets.json]: Format error: Remove the `\` symbol at `\n\mySet...`. Shuld be `\nmySet...`.

**Comment by rafaparadela** [comment]
Hi @DrMabulle. Makes sense your suggestion and your new assertions are useful for understand the immutability of the original set. But there are two format error that should be corrected before merge.
Once addressed I'll be pleased to merge it.

Thank you very much for your contribution.

**Comment by DrMabulle** [comment]
Hi @rafaparadela. Sorry about that. I made the corrections.

**Comment by rafaparadela** [comment]
So fast! ;-) Shipping the boat...

**Pull request merged**